src/py42/clients/audit_logs.py

```diff
@@ -0,0 +1,41 @@
+class AuditLogsClient(object):
+    def __init__(self, audit_log_service):
+        self._audit_log_service = audit_log_service
+
+    def get_all(
+        self,
+        begin_time=None,
+        end_time=None,
+        event_types=None,
+        user_ids=None,
+        user_names=None,
+        user_ip_addresses=None,
+        affected_user_ids=None,
+        affected_user_names=None,
+    ):
+        """Retrieve audit logs, filtered based on given arguments.
+
+        Args:
+            begin_time (datetime, optional): A datetime.datetime instance for a given timestamp. Defaults to None.
+            end_time (datetime, optional): A datetime.datetime instance for a given timestamp. Defaults to None.
+            event_types (str, optional): Comma separated str of valid event types. Defaults to None.
```

Comment:
see me not in the `Service` class, I think all these parameters should be lists instead comma separated strings

Reply:
changed.